src/test/java/TicTacToe/WinLogicTest.java

```diff
@@ -0,0 +1,122 @@
+package TicTacToe;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+public class WinLogicTest {
+    @InjectMocks
+    private WinLogic winLogic;
+    @Mock
+    private Board board;
+
+    private static final char CROSS = 'X';
+    private static final char CIRCLE = 'O';
+    private final int size = 3;
```

Comment:
(Formatting) To jest const :p

Reply:
Chodziło o tą linijkę ` private final int size = 3;`
jest naprawiona